pycbc/inference/distributions.py

```diff
@@ -1036,15 +1034,17 @@ def from_config(cls, cp, section, variable_args):
                                 + ["var-%s"%param for param in variable_args]
 
         # get input kwargs
-        low = []
-        high = []
-        mean = []
-        var = []
+        params = {}
         for param in variable_args:
-            low.append( float(cp.get_opt_tag(section, "min-%s"%param, tag)) )
-            high.append( float(cp.get_opt_tag(section, "max-%s"%param, tag)) )
-            mean.append( float(cp.get_opt_tag(section, "mean-%s"%param, tag)) )
-            var.append( float(cp.get_opt_tag(section, "var-%s"%param, tag)) )
+            params[param] = (None, None, None, None)
+            params[param][0] = float(cp.get_opt_tag(section,
+                                                   "min-%s"%param, tag))
+            params[param][1] = float(cp.get_opt_tag(section,
+                                                   "max-%s"%param, tag))
+            params[param][2] = float(cp.get_opt_tag(section,
+                                                   "mean-%s"%param, tag))
+            params[param][3] = float(cp.get_opt_tag(section,
+                                                   "var-%s"%param, tag))
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent
* [E228] missing whitespace around modulo operator

<!-- Comment by [SideCI](https://sideci.com) -->